scripts/generator/templates/template-vaadin-spring-bom.xml

```diff
@@ -43,6 +43,16 @@
                 <artifactId>vaadin-core</artifactId>
                 <version>${project.version}</version>
             </dependency>
+            <dependency>
+                <groupId>com.vaadin</groupId>
+                <artifactId>vaadin-dev-flow</artifactId>
+                <version>${project.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>com.vaadin</groupId>
+                <artifactId>vaadin-dev-hilla</artifactId>
+                <version>${project.version}</version>
+            </dependency>
```

Comment:
A new dilemma: `vaadin-dev-hilla` is inside deps management here and it has to be added to `<dependencies>` in `vaadin-spring-boot-starter` to become a part of it. The as is structure doesn't add `vaadin-dev-hilla` into Vaadin Spring projects.

But if we add it, we no more could exclude it in prod mode by excluding `vaadin-dev`.